README.md

```diff
@@ -141,6 +141,28 @@ merge_client.ticketing.tickets.create(
     ))
 ```
 
+## Pagination
+
+The SDK may return paginated results. Endpoints that return paginated results will 
+include a `next` and `prev` property on the response. To get the next page, you can 
+pass in the value of `next` to the cursor property on the request. Similarly, to 
+get the previous page, you can pass in the value of `prev` to the cursor property on 
+the request. 
+
+Below is an example of iterating overa all pages:
```

Comment:
minor typo: overall

Reply:
thanks, meant to write `iterating over all pages`